os/booting-on-google-compute-engine.md

```diff
@@ -104,6 +108,34 @@ gcloud compute ssh --zone us-central1-a core@<instance-name>
 
 Users other than `core`, which are set up by the GCE account manager, may not be a member of required groups. If you have issues, try running commands such as `journalctl` with sudo.
 
+### Using OS Login
+
+You can log in using your Google account on instances with OS Login enabled. OS Login needs to be [enabled in the GCE console](https://cloud.google.com/compute/docs/instances/managing-instance-access#enable_oslogin) and on the instance. It is enabled by default on the instance for nodes provisioned with Container Linux 1849.0.0 or later. Once enabled, you can log into your Container Linux instances using:
+
+```sh
+gcloud compute ssh --zone us-central1-a <instance-name>
+```
+
+Which will use your GCE user to log in.
+
+
+#### Disabling OS Login on newly provisioned nodes
+
+You can disable the OS Login functionality by masking the `oem-gce-enable-oslogin.service` unit:
+
+```yaml container-linux-config:gce
+systemd:
+  units:
+    - name: oem-gce-enable-oslogin.service
+      mask: true
+```
+
+It is recommended to also disable the OS Login functionality in the GCE console as well when disabling it on the instance.
+
+#### Enabling OS Login on existing nodes
+
+To enable OS Login on existing nodes, copy the files [here](link to coreos-overlay filesdir) and run [this script](link to script in filesdir). This is the script that the `oem-gce-enable-oslogin.service` unit runs. It copies pam, sudoers, and sshd configuration and changes the `/etc/nsswitch.conf` to enable the OS Login modules.
```

Comment:
Do we really want to recommend that users do this?  I'd be comfortable just saying that OS Login is supported on 1849 or higher.

Reply:
/shrug I'm fine with dropping it.